src/librustc_yk_sections/mir_cfg.rs

```diff
@@ -400,11 +401,14 @@ impl<'tcx> ToPack<ykpack::Terminator> for (&ConvCx<'_, 'tcx, '_>, &Terminator<'t
 }
 
 /// BasicBlockData -> Pack
-impl<'tcx> ToPack<ykpack::BasicBlock> for (&ConvCx<'_, 'tcx, '_>, BasicBlock, &BasicBlockData<'tcx>) {
+impl<'tcx> ToPack<ykpack::BasicBlock> for
+    (&ConvCx<'_, 'tcx, '_>, BasicBlock, &BasicBlockData<'tcx>) {
+
```

Comment:
Random blank line?

Reply:
It looks weird when a line continuation is butted up against a "real" indent.

Reply:
Put the curly on the next line then?